### ISSUE TITLE
Changing a page name gives duplicate error, Multilanguage

### ISSUE THREAD

**Issue by somatonic** [ryancramerdesign/ProcessWire#1507]
Session: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'de' for key 'path'

"de" seems to be from the home page name.

![image](https://cloud.githubusercontent.com/assets/370148/11152975/22e4d4f4-8a36-11e5-8c2a-5f49e5b5ad6a.png)

Changed page name from "www.domain.ch" to "dev.domain.ch"

**Comment by ryancramerdesign** [comment]
I'm wondering which 'path' column its referring to, since the default core install doesn't use one. I'm thinking it's from either the PagePageHistory or PagePaths module. Can you check to see if you've got either one of those installed, and if so, which one? It's also possible it's coming from a 3rd party module, though less likely. 

Also, do you have a child page of the homepage with the name 'de'?

Beyond seeing an error message, did it cause any other issues after that, or prevent you from completing the update?

**Comment by somatonic** [comment]
PagePaths is installed. No there's no other "de" under root. I figured that PagePaths is the problem in combination with Multisite module which modifies paths. 

Since PagePaths isn't working for multilanguage I'll deinstall it anyway. 

PagePathHistory also isn't multilanguage compatible I see.

**Comment by ryancramerdesign** [comment]
Did it let you complete the page rename, or did this particular error prevent the operation or interfere in any way (other than an error message)?

**Comment by somatonic** [comment]
No, it did work as far as I cant tell. Just showing the error message.
